### PR TITLE
Leaping Through Time

### PR DESCRIPTION
**Pull request by ActuallyTaylor**
Upgrades the CANOpen version to be in line with the [main](https://github.com/embedded-office/canopen-stack) repository for canopen-stack

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -22,68 +22,93 @@ add_library(canopen-stack)
...
+        # network services
+        # - CiA301
+        service/cia301/co_csdo.c
+        service/cia301/co_emcy.c
+        service/cia301/co_pdo.c
+        service/cia301/co_ssdo.c
+        service/cia301/co_sync.c
+        # - CiA305
+        service/cia305/co_lss.c
+)
```
Comment (mjmagee991) [inline src/CMakeLists.txt]: Add a newline

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -22,68 +22,93 @@ add_library(canopen-stack)
...
 target_include_directories(canopen-stack
-  PUBLIC
-    config
-    core
-    hal
-    object/basic
-    object/cia301
-    service/cia301
-    service/cia305
+        PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/service/cia305>
```
Comment (mjmagee991) [inline src/CMakeLists.txt]: Move `PUBLIC` onto the line above to match all the other includes

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Minor details

**Review by mjmagee991 — APPROVED** [review]
Nice work

**Review by ishan-renjen — APPROVED** [review]

**Pull request merged**